databricks/koalas/missing/frame.py

```diff
@@ -39,10 +39,13 @@ class _MissingPandasLikeDataFrame(object):
     asfreq = _unsupported_function("asfreq")
     asof = _unsupported_function("asof")
     at_time = _unsupported_function("at_time")
+    backfill = _unsupported_function("_backfill")
```

Comment:
`_unsupported_function("backfill")`?

Reply:
@ueshin It's removed at #1798 ! Thanks for the review :)